Scrapping a Website/main.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
```

Comment:
`pandas` is a bit of overkill here, could you please implement a solution without using it?